### PR TITLE
Adds Coder Socks

### PR DESCRIPTION
**Pull request by qwertyquerty**
## About The Pull Request

image

Sprites by kate bishop

## Why It's Good For The Game

😳 

## Changelog
:cl:
add: Coder Socks
/:cl:

### CODE REVIEW

**Review by Moccha-Bee — APPROVED** [review]

**Comment by ghost** [comment]
make them a beecoin shop item worth 100.000 beecoins.

**Comment by katebishop49** [comment]
god my fucking sides

**Review by aguyiguess — APPROVED** [review]

**Comment by qwertyquerty** [comment]
> make them a beecoin shop item worth 100.000 beecoins.

No we do not charge to be coder. Anyone is welcome to begin questioning their sexuality regardless of financial status

**Review by ghost — APPROVED** [review]

**Review by Archanial — CHANGES_REQUESTED** [review]
Delete this

**Review by francinum — APPROVED** [review]

**Review by Cenrus — APPROVED** [review]

**Comment by TeomanTheGreat** [comment]
Yes

File: code/modules/mob/dead/new_player/sprite_accessories.dm [hunk 1]
```diff
@@ -1511,6 +1511,18 @@
...
+/datum/sprite_accessory/socks/codersocks_pink
+	name = "Coder Socks (Pink)"
+	icon_state = "codersocks_pink"
+
+/datum/sprite_accessory/socks/codersocks_blue
+	name = "Coder Socks (Blue)"
+	icon_state = "codersocks_blue"
+
+/datum/sprite_accessory/socks/codersocks_trans
+	name = "Coder Socks (Trans)"
```
Comment (alexkar598) [inline code/modules/mob/dead/new_player/sprite_accessories.dm:1523]: Ah yes, Trans, my favourite color

Comment (qwertyquerty) [reply code/modules/mob/dead/new_player/sprite_accessories.dm:1523]: Do you have a better word for pink blue white blue pink

**Comment by ike709** [comment]
These are trans pride socks, not codersocks.

File: code/modules/mob/dead/new_player/sprite_accessories.dm (hunk 1 above)
Comment (Archanial) [reply code/modules/mob/dead/new_player/sprite_accessories.dm:1523]: what about pink-blue-white-blue-pink?

**Review by ike709 — CHANGES_REQUESTED** [review]
Should restrict it to ckeys in the Top Contributors list.

**Review by LastCrusader105 — APPROVED** [review]
Yes

My bad no change required, no bully pls I’m learning

**Comment by katebishop49** [comment]
@ike709  i sprited every pride flag in the original codersocks 
qwert made me

**Comment by qwertyquerty** [comment]
based ike

File: code/modules/mob/dead/new_player/sprite_accessories.dm (hunk 1 above)
Comment (Sinestia) [reply code/modules/mob/dead/new_player/sprite_accessories.dm:1523]: > 
> 
> Do you have a better word for pink blue white blue pink

pastel stripes

**Comment by ghost** [comment]
Kate what the fuck

**Comment by ToasterBan** [comment]
Jimmy Bwickets shall rise once again

**Comment by MCterra10** [comment]
austation, reporting in

**Comment by ghost** [comment]
Terra is a loooser he blocked me from au

**Pull request merged**